localstack/services/opensearch/cluster.py

```diff
@@ -156,6 +159,22 @@ def bin_name(self) -> str:
     def os_user(self):
         return constants.OS_USER_OPENSEARCH
 
+    def start(self) -> bool:
+        if not config.is_emulation_mode_full():
```

Comment:
A possible alternative would be to wrap `OpensearchCluster` in a `MockedOpensearchCluster` that inherits from it, if we prefer to keep the `OpensearchCluster` class clean of emulation type switches..

Reply:
yes we should definitely do that!